### ISSUE TITLE
Error: OpenFileSystemWithId: 102

### ISSUE THREAD

**Issue by Hauteknits** [XorTroll/Goldleaf#87]
Hi, while trying to install hbmenu.nsp and retroarch.nsp. I get this error: "failed to open file system with id: @Sdcard://tinfoil/nsp/RetroArch.nsp/(string of numbers, not sure if key or not so left out).cnmt.nca." I am running atmosphere 7.1 on v6.0.1. Am I doing something wrong?

**Comment by XorTroll** [comment]
Possibly the nsp or the cnmt nca inside it are corrupted or fucked up in some way

**Comment by LittlestCube** [comment]
Are you using Atmosphere with SigPatches? I believe I've gotten that error before when I didn't turn them on.

**Comment by GiovanH** [comment]
I am experiencing the same issue. To troubleshoot, I tested it with a game cart I dumped to XCI myself, packaged to NSP, and attempted to install. I get OpenFileSystemWithId:101, with the same message as the original post.

I actually tested dumping the XCI and reinstalling the XCI immediately with ZeroTwoXci, a fork of this project, and I get the exact same error, so I don't believe it's the NSP packaging that's the problem.
